config/routes.rb

```diff
@@ -1,10 +1,8 @@
 Rails.application.routes.draw do
   devise_for :users
   root 'users#index'
-  # get "users", to: "users#index"
-  # get "users/:id", to: "users#show"
-
-  resources :users, only: [:index, :show] do
+  
+  resources :users, only: [:index, :show, :destroy] do
     resources :posts, only: [:index, :show, :new, :create, :destroy] do
       resources :comments, only: [:new, :create, :destroy]
       resources :likes, only: [:create, :destroy]
```

Comment:
No API endpoints here that returns all post in JSON format, hence it is difficult to get the api values for all comments and posts (nothing shows for the api when you run the rails routes) . Kindly check and add accordingly.

Reply:
Hi @jerryowusu, can you help me in this regard